shared/utils/oscap-svg-support.py

```diff
@@ -29,14 +29,40 @@
 """
 
 
+def subprocess_check_output(*popenargs, **kwargs):
+    # Backport of subprocess.check_output taken from
+    # https://gist.github.com/edufelipe/1027906
+    #
+    # Originally from Python 2.7 stdlib under PSF, compatible with LGPL2+
+    # Copyright (c) 2003-2005 by Peter Astrand <astrand@lysator.liu.se>
+    # Changes by Eduardo Felipe
+
+    process = subprocess.Popen(stdout=subprocess.PIPE, *popenargs, **kwargs)
+    output, unused_err = process.communicate()
+    retcode = process.poll()
+    if retcode:
+        cmd = kwargs.get("args")
+        if cmd is None:
+            cmd = popenargs[0]
+        error = subprocess.CalledProcessError(retcode, cmd)
+        error.output = output
+        raise error
+    return output
+
+
+if hasattr(subprocess, "check_output"):
+    # if available we just use the real function
+    subprocess_check_output = subprocess.check_output
+
+
 def main():
     oscap_executable = sys.argv[1]
 
     xccdf = tempfile.NamedTemporaryFile()
     xccdf.write(svg_benchmark.encode("utf-8"))
     xccdf.flush()
 
-    out = subprocess.check_output(
+    out = subprocess_check_output(
         [oscap_executable, "xccdf", "generate", "guide", xccdf.name]
```

Comment:
We could do that but IMO this is better. Having a function that was written for Python 2.7 that we don't have to maintain is fine IMO.